### PR TITLE
[DOCS-49] Move Glossary

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Glossary | https://deploy-preview-265--cobalt-docs.netlify.app/glossary/ | Moved to Platform Deep Dive |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Getting started/Pentest objectives/stack.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ technologies can vary by asset:
 When you create or update a Web, Mobile, API, or combined [asset](/getting-started/assets/), you can [add a technology stack](/platform-deep-dive/assets/risk-advisories/#add-a-technology-stack-for-your-asset) for it. Technologies that you selected from a predefined list on the asset details page populate in the **Technology Stack** field in the pentest wizard.
 
 - You can add more technologies for your pentest, in addition to those that you specified for your asset earlier.
-- We don't show a predefined list of technologies in the **Technology Stack** list in the pentest wizard, as opposed to the asset details page. Type a technology, and press Enter to confirm.
+- We don't show a predefined list of technologies in the **Technology Stack** list in the pentest wizard, as opposed to the asset details page. Type a technology, and press <kbd>Enter</kbd> to confirm.
```
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/Pentest objectives/stack.md:22]: @mjang-cobalt Experimenting with the <kbd> tag in several files.

**Comment by mjang-cobalt** [comment]
We've needed to move this out from the Getting Started guide for some time. IMO, this should be in a dedicated book. IOW, we shouldn't "hide" it inside any other book. ref: internal Jira DOCS-49

File: content/en/Getting started/Pentest objectives/stack.md (hunk 1 above)
Comment (mjang-cobalt) [reply content/en/Getting started/Pentest objectives/stack.md:22]: I love your willingness to experiment. However, I haven't seen this kind of graphic with keyboard entries before. I'd rather not set up a new design pattern in our docs. 

OTOH, if you can show me other software docs that use this graphic for keyboard entries -- of if you have the support of our designers, then let's put this into our style guide.

Screenshot 2023-01-05 at 6 29 23 AM

Comment (ana-dashuk-cobalt) [reply content/en/Getting started/Pentest objectives/stack.md:22]: @mjang-cobalt I saw this recommendation in the Google Developer Style Guide: https://developers.google.com/style/ui-elements#keyboard

I agree that it doesn't look perfect when rendered, so I'm removing this tag for the time being. I'll keep this in mind for our (future) brainstorming sessions about the portal look and feel.

**Comment by ana-dashuk-cobalt** [comment]
@mjang-cobalt I moved the Glossary to a dedicated book.

**Pull request closed without merging**